src/sys/signal.rs

```diff
@@ -3,7 +3,9 @@
 
 use libc;
 use {Errno, Error, Result};
+#[cfg(not(target_os = "openbsd"))]
 use std::fmt;
```

Comment:
These can be combined into one `use` statement like `use std::fmt::{self, Debug};`.